docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -52,9 +52,9 @@ components:
       command: ['sleep', 'infinity']
 ----
 +
-It specifies the type of the component, `container` and the `image` attribute names the image to be used for the component using the usual Docker naming conventions, that is, the above `image` attribute is equal to `docker.io/library/golang:latest`.
+Specify the type of component with the `container`. Use the `image` attribute to name the image used for the component. When naming the `image` use Docker naming conventions; for example, the `image` attribute is equal to `docker.io/library/golang:latest`.
 +
-A `container` component has many features that enable augmenting the image with additional resources and information needed for meaningful integration of the *tool* provided by the image with {prod}.
+A `container` component has many features so you can augment the image with additional resources and information needed for integration of the tools provided by the devfile image.
```

Comment:
I feel like this wording is slightly confusing. Maybe smth along the lines of "With a `container component, you can augment..."